deploy/stacks/container.py

```diff
@@ -44,11 +47,20 @@ def __init__(
         self._ecr_repository = ecr_repository
         self._vpc = vpc
         self._prod_sizing = prod_sizing
+        self._email_custom_domain = email_custom_domain
+        self._ses_configuration_set = ses_configuration_set
 
         (self.scheduled_tasks_sg, self.share_manager_sg) = self.create_ecs_security_groups(
             envname, resource_prefix, vpc, vpce_connection, s3_prefix_list, lambdas
         )
         self.ecs_security_groups: [aws_ec2.SecurityGroup] = [self.scheduled_tasks_sg, self.share_manager_sg]
+        self.env_vars = self._create_env('INFO')
```

Comment:
Maybe we can move this logic inside the creation of the task in `add_persistent_email_reminders_task` and make it a local variable inside the function

Reply:
We could, but environment variables are usually defined in __init__ right? I dont like the idea so much of it being moved inside the add_persistent_email_reminders_task function

Reply:
Convinced :)